pkg/security/resolvers/tags/resolver.go

```diff
@@ -20,6 +22,14 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+// Event defines the tags event type
+type Event int
+
+const (
+	// WorkloadSelectorResolved is used to notify that a new cgroup with a resolved workload selector is ready
+	WorkloadSelectorResolved Event = iota
+)
```

Comment:
Shouldn't we simplify the code, since now there is only one remaining event value?

Reply:
I'm not a fan of having a "RegisterListener" function that triggers the callback for an event type that you don't know about. What I could do is move this listener this to pkg/security/utils, WDYT ?